cluster/centos/node/bin/mk-docker-opts.sh

```diff
@@ -40,8 +40,9 @@ combined_opts_key="DOCKER_OPTS"
 indiv_opts=false
 combined_opts=false
 ipmasq=true
+val=""
 
-while getopts "f:d:ick:" opt; do
+while getopts "f:d:icmk:" opt; do
```

Comment:
do you know if this worked at all before by any chance? 😱

Reply:
I didn’t see the script called with that specific option in the quick search I did, but if it is called somewhere....my guess is it would not be work 😕